src/controllers/employer.ts

```diff
@@ -65,6 +65,44 @@ export const registerEmployer = async (
   }
 }
 
+//**Update employer*/
+export const updateEmployer = async (
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  try {
+    const employerId = req.params.id
+    const { update } = req.body
+    const employer = await Employer.findOne({ where: { id: employerId } })
```

Comment:
you can shorten this, and you have to load the relation also if you want to update the `credential`: 
`await Employer.findOne(employerId, {relations: ['credentials']})`